### DISCUSSION TITLE
Security onion 2.4.30, it works on airgap

### DISCUSSION THREAD

**Discussion by Sabrine-se** [Security-Onion-Solutions/securityonion#12035] [Unsupported Versions] [answered]
Security onion 2.4.30, it works on airgap 
i want to install this version 2.4.30 
aigap
two nodes search and manager without forward no monitoring just collecting logs from firewall
i want to know if this version works on airgap with 2 nodes search and manager

**Comment by dougburks** [comment, marked as answer]
Yes, Security Onion 2.4.30 works on airgap networks:
https://docs.securityonion.net/en/2.4/airgap.html
